pydis_site/templates/home/timeline.html

```diff
@@ -19,8 +19,8 @@
 
         <div class="cd-timeline__content text-component">
           <h2>Python Discord is created</h2>
-          <p class="color-contrast-medium"><strong>joe</strong> becomes one of the owners around 3 days after it
-            is created, and <strong>lemon</strong> joins the owner team later in the year, when the community
+          <p class="color-contrast-medium"><strong>Joe Banks</strong> becomes one of the owners around 3 days after it
+            is created, and <strong>Leon Sandøy</strong> joins the owner team later in the year, when the community
```

Comment:
```suggestion
            is created, and <strong>Leon Sandøy</strong> (lemon) joins the owner team later in the year, when the community
```
What do you think about putting the nicknames inside parentheses, so people will be recognized on the server? This would go for all people mentioned.

Reply:
I was actually doing that earlier, I just removed it. I don't know if it's necessary for joe and sebastiaan since their names are their nicknames.

Reply:
Yeah, that's true. I think it would be nice nice to have for you, though.

Reply:
Okay, added! 👍🏼